tests/storage/mpathconf_test.py

```diff
@@ -49,6 +52,32 @@
 }
 """
 
+FAKE_MPATHD_OUTPUT = Template("""\
+defaults {
+        verbosity 2
+        polling_interval 5
+        multipath_dir "/lib64/multipath"
+        path_selector "service-time 0"
+        prio_args ""
+        user_friendly_names $defaults_value
+        find_multipaths_timeout -10
+        enable_foreign "NONE"
+}
+overrides {
+        user_friendly_names $overrides_value
+        no_path_retry 16
+}
+multipaths {
+    multipath {
+        wwid "wwid1"
+        user_friendly_names $mpath_value
+    }
+    multipath {
+        wwid "wwid2"
+    }
```

Comment:
You need to test also "devices" section.

Reply:
I'll use generic names instead.